Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -481,6 +481,12 @@ private string FindMGCB()
 
         private void DoBuild(string commands)
         {
+            if(!Environment.Is64BitOperatingSystem)
+            {
+                View.OutputAppend("Build failed:  Content can only be compiled from a 64 bit OS.");
```

Comment:
> Maybe this is better at the start of MGCB.exe ?

We are compiling it for 64 bit: https://github.com/mono/MonoGame/blob/develop/Build/Projects/MGCB.definition#L30 so its not really even runnable from 32 bit.